internal/metamorph/store/postgresql/migrations/000016_multiple_callbacks.down.sql

```diff
@@ -0,0 +1,13 @@
+-- Step 1: Add the old 'callback_url' and 'callback_token' columns back
+ALTER TABLE metamorph.transactions
+    ADD COLUMN callback_url TEXT,
+    ADD COLUMN callback_token TEXT;
+
+-- Step 2: Populate 'callback_url' and 'callback_token' with the first object in the 'callbacks' JSON array
+UPDATE metamorph.transactions
+SET callback_url = (callbacks->0->>'callback_url'),
+    callback_token = (callbacks->0->>'callback_token');
```

Comment:
nit: In order not to have this security warning: https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=539&types=BUG&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&inNewCodePeriod=true&sinceLeakPeriod=true&id=bitcoin-sv_arc

add something like
```
WHERE jsonb_array_length(callbacks) > 0
```

Reply:
Done